package-lock.json

```diff
@@ -4022,6 +4022,23 @@
           "integrity": "sha512-EykJT/Q1KjTWctppgIAgfSO0tKVuZUjhgMr17kqTumMl6Afv3EISleU7qZUzoXDFTAHTDC4NOoG/ZxU3EvlMPQ==",
           "dev": true
         },
+        "html-webpack-plugin": {
```

Comment:
this isn't in package.json, it is here because it is a dependency of one of our dependencies.